package.json

```diff
@@ -10,59 +10,53 @@
+    "lint": "standard | notify-error",
+    "make-mock-compiler": "node ci/makeMockCompiler.js",
+    "onchange": "onchange build/app.js -- npm run lint",
+    "prepublish": "npm-run-all -ls downloadsolc make-mock-compiler build",
+    "selenium": "selenium-standalone start",
     "serve": "http-server .",
-    "test": "standard; babel src --out-dir babelify-src; node test/index.js"
+    "start": "npm-run-all -lpr serve watch onchange",
+    "test": "standard; babel src --out-dir babelify-src; node test/index.js",
+    "watch": "mkdirp build; watchify src/index.js -p [ browserify-livereload --host 127.0.0.1 --port 1337 ] -t [ babelify --presets [ es2015 ] --no-sourceMaps ] -dv -o build/app.js"
   },
   "devDependencies": {
     "async": "^2.1.2",
-    "babel-cli": "^6.16.0",
     "babel-eslint": "^7.1.1",
-    "babel-plugin-check-es2015-constants": "^6.8.0",
-    "babel-plugin-transform-es2015-arrow-functions": "^6.8.0",
-    "babel-plugin-transform-es2015-block-scoped-functions": "^6.8.0",
-    "babel-plugin-transform-es2015-block-scoping": "^6.18.0",
-    "babel-plugin-transform-es2015-classes": "^6.18.0",
-    "babel-plugin-transform-es2015-computed-properties": "^6.8.0",
-    "babel-plugin-transform-es2015-destructuring": "^6.18.0",
-    "babel-plugin-transform-es2015-duplicate-keys": "^6.8.0",
-    "babel-plugin-transform-es2015-for-of": "^6.18.0",
-    "babel-plugin-transform-es2015-function-name": "^6.9.0",
-    "babel-plugin-transform-es2015-literals": "^6.8.0",
-    "babel-plugin-transform-es2015-object-super": "^6.8.0",
-    "babel-plugin-transform-es2015-parameters": "^6.18.0",
-    "babel-plugin-transform-es2015-shorthand-properties": "^6.18.0",
-    "babel-plugin-transform-es2015-spread": "^6.8.0",
-    "babel-plugin-transform-es2015-sticky-regex": "^6.8.0",
-    "babel-plugin-transform-es2015-template-literals": "^6.8.0",
-    "babel-plugin-transform-es2015-unicode-regex": "^6.11.0",
-    "babel-plugin-transform-regenerator": "^6.16.1",
-    "babel-polyfill": "^6.22.0",
+    "babel-preset-es2015": "^6.22.0",
     "babelify": "^7.3.0",
+    "babili": "0.0.11",
     "brace": "^0.8.0",
     "browserify": "^13.0.0",
+    "browserify-livereload": "^1.0.9",
+    "csjs-inject": "^1.0.1",
     "csslint": "^1.0.2",
-    "ethereum-remix": "https://github.com/ethereum/remix",
+    "ethereum-remix": "git+https://github.com/ethereum/remix.git",
     "ethereumjs-abi": "^0.6.4",
     "ethereumjs-block": "^1.2.2",
     "ethereumjs-tx": "^1.1.1",
     "ethereumjs-util": "^4.4.0",
     "ethereumjs-vm": "^2.0.1",
     "fast-async": "^6.1.2",
     "http-server": "0.9.0",
-    "jquery": "^2.2.0",
+    "jquery": "^3.1.1",
     "js-base64": "^2.1.9",
+    "mkdirp": "^0.5.1",
     "nightwatch": "^0.9.3",
+    "notify-error": "^1.2.0",
+    "npm-run-all": "^4.0.2",
+    "onchange": "^3.2.1",
     "solc": "https://github.com/ethereum/solc-js",
-    "standard": "^8.5.0",
+    "standard": "^8.6.0",
```

Comment:
it is not necessary to change the version I think (same for jquery, yo-yo)
@axic @chriseth opininon on that ?